content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -264,10 +264,13 @@ running in a second cluster. Before you begin:
     {{< /tip >}}
 
     If the global services have actual VIPs, you can use those, but otherwise we suggest
-    using IPs from the multicast range `224.0.0.0/4` that are not already allocated.
-    These IPs are not loopback addresses and are non-routable outside of a pod.
+    using IPs from the class E addresses range `240.0.0.0/4` that are preserved.
+    These IPs are not loopback addresses and are non-routable outside of a pod.    
```

Comment:
Maybe delete this sentence and add to the Note below?

Reply:
ok